src/bvals/bvals_base.cpp

```diff
@@ -100,8 +100,8 @@ BoundaryBase::BoundaryBase(Mesh *pm, LogicalLocation iloc, RegionSize isize,
   if (pmy_mesh_->multilevel) { // SMR or AMR
     // allocate surface area array
     int nc1 = block_size_.nx1 + 2*NGHOST;
-    sarea_[0].NewAthenaArray(nc1);
-    sarea_[1].NewAthenaArray(nc1);
+    sarea_[0].NewParArrayND(nc1);
+    sarea_[1].NewParArrayND(nc1);
```

Comment:
We should move towards naming these guys when we do this. Obviously that's something that will take a while to fix.

Reply:
replaced all of these calls everywhere.